packages/gatsby/src/schema/schema.js

```diff
@@ -689,6 +703,40 @@ const addCustomResolveFunctions = async ({ schemaComposer, parentSpan }) => {
   })
 }
 
+const determineSearchableFields = ({ schemaComposer, typeComposer }) => {
```

Comment:
Currently we make everything searchable and resolvable. However in future we might change that behaviour, so that weird things like `fixed` in sharp aren't searchable.